ontology/yaml/resources/ELECTRICAL/entity_types/ABSTRACT.yaml

```diff
@@ -133,6 +152,16 @@ BTEPM:
   - battery_current_sensor
   - line_frequency_sensor  
 
+SBATM:
+  description: "Simple battery monitoringy typically used with UPS"
+  is_abstract: true
+  uses:
+  - battery_charge_percentage_sensor
+  - batterycapacity_sensor
+  - battery_status
+  - battery_test_status
+  - failed_battery_alarm
```

Comment:
`alarm` fields should always be `opt_uses`

Reply:
fixed thank you